components/posts/posts.php

```diff
@@ -82,9 +82,9 @@ function mm_posts( $args ) {
 	);
 
 	// Add to our query if additional params have been passed.
-	if ( $post_ids ) {
+	if ( $post_titles ) {
 
-		$query_args['post__in'] = $post_ids;
+		$query_args['post__in'] = $post_titles;
 		$query_args['orderby']  = 'post__in';
```

Comment:
According to the docs for WP_Query, `post__in` only accepts post IDs, and I see that the new autocomplete field is saving post IDs, so I think the variable in this function should remain $post_ids. If you wanted to accept `post_slugs` and/or `post_titles` as separate params from `post_ids`, I'm all for it, but I think keeping a clear path to specifying just `post_ids` is important, especially for the non-VC use cases.